### PR TITLE
Make the script work in a FreeBSD jail, in i.e. FreeNAS

### PR DESCRIPTION
**Pull request by SebastianChudoba**
I've made that shell-script to work in a FreeNAS jail, for i.e. test an openvpn installation.
Command "shuf" does not exist in BSD so I made use of "jot".

### CODE REVIEW

File: dnsleaktest_FreeBSDjail.sh [hunk 1]
```diff
@@ -41,7 +45,10 @@ program_exit ping
 program_exit jq
 check_internet_connection
 
-id=$(shuf -i 1000000-9999999 -n 1)
+#id=$(shuf -i 1000000-9999999 -n 1)
```
Comment (macvk) [inline dnsleaktest_FreeBSDjail.sh:48]: I think you should use the following code. what you think?

if hash shuf 2>/dev/null; then
    id=$(shuf -i 1000000-9999999 -n 1)
else
    id=$(jot -w %i -r 1 1000000 9999999)
fi

File: dnsleaktest_FreeBSDjail.sh [hunk 2]
```diff
@@ -1,4 +1,8 @@
-#!/bin/bash
```
Comment (macvk) [inline dnsleaktest_FreeBSDjail.sh:1]: Please use the following code

`#!/usr/bin/env bash
`

please test the suggested code in your environment.

**Comment by macvk** [comment]
Please do not rename the file dnsleaktest.sh

File: dnsleaktest_FreeBSDjail.sh (hunk 1 above)
Comment (SebastianChudoba) [reply dnsleaktest_FreeBSDjail.sh:48]: This works well, withouth having to rename the dnsleaktest.sh ;)

File: dnsleaktest_FreeBSDjail.sh (hunk 2 above)
Comment (SebastianChudoba) [reply dnsleaktest_FreeBSDjail.sh:1]: Yes, that's better. Confirmed working a freebsd jail with pkg bash installed.

**Pull request merged**